front/app/containers/App/Meta.tsx

```diff
@@ -35,11 +35,20 @@ const Meta = () => {
 
     // TODO: Fix this the next time the file is edited.
     // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-    const headerBg = homepageLayout.data.attributes.craftjs_json
-      ? Object.values(homepageLayout.data.attributes.craftjs_json).find(
-          (node) => node.displayName === 'HomepageBanner'
-        )?.props.image?.imageUrl
-      : '';
+    const bannerNode = homepageLayout?.data?.attributes?.craftjs_json
+      ? Object.keys(homepageLayout.data.attributes.craftjs_json).reduce(
+          (result, nodeId) => {
+            const node = homepageLayout.data.attributes.craftjs_json[nodeId];
+            if (node.displayName === 'HomepageBanner') {
```

Comment:
Hmm was the fix not to replace `displayName` by `resolvedName`? I also think that using `find` is a more logical choice than `reduce` in this context, even though they accomplish the same thing